src/precomputed-requestor.ts

```diff
@@ -19,6 +28,14 @@ export default class PrecomputedFlagRequestor {
       subject_attributes: this.subjectAttributes,
     });
 
+    if (this.onPrecomputedResponse) {
+      this.onPrecomputedResponse({
+        decodedSalt: precomputedResponse?.salt ? decodeBase64(precomputedResponse.salt) : undefined,
```

Comment:
Also wondering if it matters is the decoded salt is represented as a string or an array. 

I see in Rust that the salt's [unencoded representation](https://github.com/Eppo-exp/eppo-multiplatform/blob/d8dd6e5a6662dcc2e65177679df4aa5f2790de01/eppo_core/src/precomputed.rs#L72) is `[u8; 16]`, i.e. array of 16 8-bit integers. Whereas in JS we're passing it around as a string, though I don't know if that matters. There is a `base64.toUint8Array` we could use, but also `Array.from(decodedSalt).map((char) => char.charCodeAt(0)` where `decodedSalt` is the string returned by `decodeBase64` is an identical array.